src/http/fallback_template.eml.ml

```diff
@@ -0,0 +1,81 @@
+let css = Dream__asset.css
+
+let favicon = Base64.encode Dream__asset.favicon |> Result.get_ok
+
+let render ~code ~reason = 
+  <!DOCTYPE html>
+  <html lang="en">
+    <head>
+      <meta charset="utf-8" />
+      <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+      <link rel="icon" type="image/png" sizes="16x16" href="data:image/png;base64,<%s! favicon %>" />
+      <link rel="stylesheet" href="https://rsms.me/inter/inter.css" />
+      <style>
+        <%s! css %>
+      </style>
+      <title><%i code %> <%s reason %></title>
+    </head>
+
+    <body class="antialiased">
+      <div class="py-4 sm:py-12">
+        <div class="max-w-8xl mx-auto px-4 sm:px-6 lg:px-8">
+          <h2 class="text-2xl leading-8 font-semibold font-display text-gray-900 sm:text-3xl sm:leading-9">
+            Oops, an error occured!
+          </h2>
+          <div class="mt-0 mb-4 text-gray-600">
+            <%i code %> <%s reason %>
+          </div>
+        </div>
+      </div>
+    </body>
+  </html>
+
+let render_debug ~debug_dump ~code ~reason = 
+  <!DOCTYPE html>
+  <html lang="en">
+    <head>
+      <meta charset="utf-8" />
+      <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+      <link rel="icon" type="image/png" sizes="16x16" href="data:image/png;base64,<%s! favicon %>" />
+      <link rel="stylesheet" href="https://rsms.me/inter/inter.css" />
+      <style>
+        <%s! css %>
+      </style>
+      <title><%i code %> <%s reason %></title>
+    </head>
+
+    <body class="antialiased">
```

Comment:
Given how relatively simple the error template is, can you remove Tailwind in favor of just plain CSS? The CSS can probably just be written inline, it's quite brief. It looks like together with not overriding the app's favicon, that will simplify away the `asset` directory.